components/session-timeout-warning/Readme.md

```diff
@@ -0,0 +1,43 @@
+# Session Timeout Warning Behaviour
+
+## What this does
+Makes the content related to the session timeout warning behaviour customisable. This includes the content in the session timeout warning dialog and on the exit page after the user clicks to exit the form from the session timeout warning dialog.
+
+The component should be set in the project's hof.settings.json
```

Comment:
```suggestion
## What is it?
This feature allows you to customize the content related to the session timeout warning, including the messages displayed in the session timeout warning dialog and on the exit page after a user exits the form due to a session timeout.

## Enabling customisation

To enable and customize the session timeout behavior, you need to set the component in your project's `hof.settings.json` file:
```
```

Reply:
I have updated this with your content suggestions (can now be found in main readme)

Reply:
nice work